sys-apps/plocate/plocate-1.1.5.ebuild

```
@@ -0,0 +1,60 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit linux-info meson systemd
+
+DESCRIPTION="Posting locate is a much faster locate"
+HOMEPAGE="https://plocate.sesse.net/"
+
+if [[ "${PV}" == "9999" ]]; then
+	inherit git-r3
+	EGIT_REPO_URL="https://git.sesse.net/plocate"
+else
+	SRC_URI="https://plocate.sesse.net/download/${P}.tar.gz"
+fi
+
+LICENSE="GPL-2 GPL-2+"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="io-uring"
+
+RDEPEND="
+	acct-group/locate
+	app-arch/zstd:=
+	!sys-apps/mlocate
+	io-uring? ( sys-libs/liburing:= )
+"
+DEPEND="${RDEPEND}"
+
+BDEPEND="acct-group/locate"
+
+CONFIG_CHECK="~IO_URING"
+ERROR_IO_URING="recommended for USE=io-uring"
+
+PATCHES=(
+	"${FILESDIR}"/plocate-1.1.5-no-systemd-check.patch
+)
+
+src_prepare() {
+	# pretend liburing dep doesn't exist when USE flag off
+	use io-uring || sed -i "s/dependency('liburing/dependency('/" meson.build || die
```

```suggestion
	if ! use io-uring ; then
	    sed -i "s/dependency('liburing/dependency('/" meson.build || die
	fi
```
(Just a style thing on my part, not required per se).